### PR TITLE
[Improvement][K8S] Remove ResourceQuota

### LINKED ISSUE #14990: [Improvement][K8S] Remove ResourceQuota
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

- https://github.com/apache/dolphinscheduler/issues/14593
- https://github.com/apache/dolphinscheduler/pull/14597#issuecomment-1643387666

As mentioned above, we have removed the ability to directly manage Kubernetes namespaces and instead only establish association mappings. Based on this, it would be better to directly remove the capability to manage resource quotas.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Gallardot**
## Purpose of the pull request

- close #14990 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/K8sNamespaceController.java [hunk 1]
```diff
@@ -104,56 +101,21 @@ public Result queryNamespaceListPaging(@Parameter(hidden = true) @RequestAttribu
...
-    @Operation(summary = "updateK8sNamespace", description = "UPDATE_NAMESPACE_NOTES")
-    @Parameters({
-            @Parameter(name = "id", description = "K8S_NAMESPACE_ID", required = true, schema = @Schema(implementation = int.class, example = "100")),
-            @Parameter(name = "userName", description = "OWNER", required = false, schema = @Schema(implementation = String.class)),
-            @Parameter(name = "limitsCpu", description = "LIMITS_CPU", required = false, schema = @Schema(implementation = double.class)),
-            @Parameter(name = "limitsMemory", description = "LIMITS_MEMORY", required = false, schema = @Schema(implementation = int.class))})
-    @PutMapping(value = "/{id}")
-    @ResponseStatus(HttpStatus.CREATED)
-    @ApiException(UPDATE_K8S_NAMESPACE_ERROR)
-    public Result updateNamespace(@Parameter(hidden = true) @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
```
Comment (Radeity) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/K8sNamespaceController.java:149]: May I ask why you remove this?

Comment (Gallardot) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/K8sNamespaceController.java:149]: This API only modifies ResourceQuota, but it is no longer necessary as there are currently no ResourceQuota, and the UI will not call this API again.

File: dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/mysql/dolphinscheduler_ddl.sql [hunk 2]
```diff
@@ -14,3 +14,90 @@
...
+    THEN
+ALTER TABLE t_ds_k8s_namespace DROP COLUMN limits_cpu;
+END IF;
+END;
+d//
+delimiter ;
+CALL drop_t_ds_k8s_namespace_col_code;
+DROP PROCEDURE drop_t_ds_k8s_namespace_col_code;
+-- ALTER TABLE t_ds_k8s_namespace DROP COLUMN IF EXISTS limits_memory;
+drop PROCEDURE if EXISTS drop_t_ds_k8s_namespace_col_code;
```
Comment (Radeity) [inline dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/mysql/dolphinscheduler_ddl.sql]: Please define unique PROCEDURE name, different from previous one.

Comment (Gallardot) [reply dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/mysql/dolphinscheduler_ddl.sql]: Done. PTAL.

**Review by Radeity — APPROVED** [review]
LGTM

**Pull request merged**